tasks/RedHat/generic/configure-systemd-tmpfiles.yml

```diff
@@ -0,0 +1,26 @@
+---
+# tasks file for sap-preconfigure: configuration
+
+- debug:
+    msg: "imported RedHat/generic/configure-systemd-tmpfiles.yml"
+
+- name: Test if file /usr/lib/tmpfiles.d/sap.conf exists
+  stat:
+    path: /usr/lib/tmpfiles.d/sap.conf
+  register: systemd_sap_conf
+  failed_when: no
+
+- debug:
+    var: systemd_sap_conf.stat.exists
+
+- name: Copy file /etc/tmpfiles.d/sap.conf if necessary
+  copy:
+    src: etc/tmpfiles.d/sap.conf
+    dest: /etc/tmpfiles.d/sap.conf
+    owner: root
+    group: root
+    mode: '0644'
+    backup: yes
+  when: not systemd_sap_conf.stat.exists
```

Comment:
You could do the same thing without the `stat` task using the `copy` module with `force: no` - https://docs.ansible.com/ansible/latest/modules/copy_module.html#copy-module

Reply:
Many thanks! I expect to be able to make this even simpler (not longer having to check for the existence of /usr/lib/tmpfiles.d/sap.conf), so I'll change this once we have an agreement with the systemd maintainer (systemd in RHEL 7 is providing /usr/lib/tmpfiles.d/sap.conf).

Reply:
Solved, we no longer check for /usr/lib/tmpfiles.d/sap.conf, see https://github.com/linux-system-roles/sap-preconfigure/pull/64/commits/b3c7a99a0ac1ceff9b4b5edddfe07b88e176ae5a .